### PR TITLE
[vs][multi-asic]: Add new platforms for multi-asic vs.

### PR DESCRIPTION
**Pull request by SuvarnaMeenakshi**
#### Why I did it
End goal: To have azure pipeline job to run multi-asic VS tests.
Intermediate goal: Require multi-asic KVM image so that the test can be run. 
The difference between single asic and multi-asic KVM image is asic.conf file which has different NUM_ASIC values.
Idea behind the approach in this PR to attain the intermediate goal above:
- Ease of building multi-asic KVM image so that any user or azure pipeline can use a simple make command to generate single or multi-asic KVM images as required.
- Use a single onie installer image and multiple KVM images for single and multi-asic images.

Current scenario:
For VS platform, sonic-vs.bin is generated which is the onie installer image and sonic-vs.img.gz is generated which is the KVM iamge.
Scenario to be achieved:
sonic-vs.bin - which will include  single asic platform,  4 asic platform and 6 asic platform.
sonic-vs.img.gz - single asic KVM image
sonic-4asic-vs.img.gz  - 4 asic KVM image
sonic-6asic-vs.img.gz - 6 asic KVM image

In this PR, 2 new platforms are added for 4-asic and 6-asic VS.

#### How I did it
- Create 4-asic and 6-asic device directories with the required files and hwsku files.
- Add onie-recovery image information in vs platform.
#### How to verify it
After this PR change, no build change.
sonic-vs.bin onie installer image should include information of new multi-asic vs platforms.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: platform/vs/onie.mk [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ONIE_RECOVERY_IMAGE = onie-recovery-x86_64-kvm_x86_64-r0.iso
 $(ONIE_RECOVERY_IMAGE)_URL = "https://sonicstorage.blob.core.windows.net/packages/onie/onie-recovery-x86_64-kvm_x86_64-r0.iso?sv=2015-04-05&sr=b&sig=XMAk1cttBFM369CMbihe5oZgXwe4uaDVfwg4CTLT%2F5U%3D&se=2155-10-13T10%3A40%3A13Z&sp=r"
-
-SONIC_ONLINE_FILES += $(ONIE_RECOVERY_IMAGE)
+ONIE_RECOVERY_MKVM_4ASIC_IMAGE = onie-recovery-x86_64-mkvm_x86_64_4_asic-r0.iso
+$(ONIE_RECOVERY_MKVM_4ASIC_IMAGE)_URL=""
+ONIE_RECOVERY_MKVM_6ASIC_IMAGE = onie-recovery-x86_64-mkvm_x86_64_6_asic-r0.iso
+$(ONIE_RECOVERY_MKVM_6ASIC_IMAGE)_URL=""
+SONIC_ONLINE_FILES += $(ONIE_RECOVERY_IMAGE) $(ONIE_RECOVERY_MKVM_4ASIC_IMAGE) $(ONIE_RECOVERY_MKVM_6ASIC_IMAGE)
```
Comment (judyjoseph) [inline platform/vs/onie.mk:7]: I see the ONIE_RECOVERY_IMAGE used in the mk file --> platform/vs/kvm-image.mk. Is that used ?
platform/vs/kvm-image.mk:$(SONIC_KVM_IMAGE)_FILES = $(ONIE_RECOVERY_IMAGE)

File: platform/vs/onie.mk [hunk 2]
```diff
@@ -1,4 +1,7 @@
 ONIE_RECOVERY_IMAGE = onie-recovery-x86_64-kvm_x86_64-r0.iso
 $(ONIE_RECOVERY_IMAGE)_URL = "https://sonicstorage.blob.core.windows.net/packages/onie/onie-recovery-x86_64-kvm_x86_64-r0.iso?sv=2015-04-05&sr=b&sig=XMAk1cttBFM369CMbihe5oZgXwe4uaDVfwg4CTLT%2F5U%3D&se=2155-10-13T10%3A40%3A13Z&sp=r"
-
-SONIC_ONLINE_FILES += $(ONIE_RECOVERY_IMAGE)
+ONIE_RECOVERY_MKVM_4ASIC_IMAGE = onie-recovery-x86_64-mkvm_x86_64_4_asic-r0.iso
+$(ONIE_RECOVERY_MKVM_4ASIC_IMAGE)_URL=""
```
Comment (judyjoseph) [inline platform/vs/onie.mk:4]: to update URL once you got to upload the file.

**Review by abdosi — DISMISSED** [review]

File: platform/vs/onie.mk (hunk 1 above)
Comment (SuvarnaMeenakshi) [reply platform/vs/onie.mk:10]: Updated platform/vs/kvm-image.mk, it is getting used, without this update the new recovery iso files will not get downloaded.

File: platform/vs/onie.mk (hunk 2 above)
Comment (SuvarnaMeenakshi) [reply platform/vs/onie.mk]: Updated URL

**Review by judyjoseph — DISMISSED** [review]

**Review by abdosi — DISMISSED** [review]

**Review by lguohan — DISMISSED** [review]

**Comment by SuvarnaMeenakshi** [comment]
> i think we do not need mkvm, we can use kvm in the onie platform string. this is a minor comments.

Modified mkvm -> kvm as suggested.

**Review by judyjoseph — APPROVED** [review]

**Pull request merged**